### PR TITLE
adapt code for -save-all option with mutliple runs

### PR DESCRIPTION
**Pull request by mszinte**
@ecasimiro maybe you can have a look at that, it seems to work on the exception of the saving of the single run components of the denoising (see line 102 in noise_model.py)... I also deleted the division by 1000 of the tr and fix one or two error there and there on which the code was blocking.

### CODE REVIEW

File: pybest/preproc.py [hunk 1]
```diff
@@ -79,10 +79,10 @@ def _run_func_parallel(ddict, cfg, run, func, logger):
     if 'fs' in cfg['space']:  # assume gifti
         if cfg['iscifti'] == 'y':
             data, tr = load_and_split_cifti(func, cfg['atlas_file'],cfg, cfg['left_id'], cfg['right_id'], cfg['subc_id'], cfg['mode'])
-            tr /= 1000 # defined in msec
+            #tr /= 1000 # defined in msec
         else:
             data, tr = load_gifti(func, cfg, return_tr=True)
-            tr /= 1000  # defined in msec
```
Comment (ecasimiro) [inline pybest/preproc.py:85]: The gifti part is not mine. I think gifti does return TR in msec and needs the /= 1000

Comment (mszinte) [reply pybest/preproc.py:85]: At the end after checking, it turns out that taking out completely these line was fine as it has to come out in seconds and load_gifti does good at that. So i'm now not changing the tr at all and it works just fine. Sorry for the confusion.

**Review by ecasimiro — APPROVED** [review]
Thanks for fixing the issue :)

**Comment by ecasimiro** [comment]
@lukassnoek I think this is ready to be merged :)

**Comment by lukassnoek** [comment]
Cool! I added the `/= 1000` because I think the TR was defined in msec in gifti files (which I rarely use anyway). Thanks for working on this @ecasimiro and @mszinte !

**Pull request merged**